templates/index.html

```diff
@@ -2,17 +2,97 @@
+                                    </form>
+                                {% elif item.status != 'Not Started' %}
+                                    <span class="badge badge-success badge-pill red">{{ item.status }}</span>
+                                {% endif %}
+                            </div>
+                            <div class="col-2">
+                                <form id="delete_item" method="post" action="">
+                                    <a href="#">
+                                        <span onclick="delete_item({{ item.id }})" class="float-right">
+                                            <i class="fa fa-trash" aria-hidden="true"></i>
+                                        </span>
+                                    </a>
+                                </form>
+                            </div>
+                        </div>
+                    {% endfor %}
+                {% else %}
+                    <div><p>No items found</p></div>
+                {% endif %}
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-10">
+                <button type="button" class="btn btn-primary float-right" data-toggle="modal"
+                        data-target="#modalAddItemForm">Add
+                    Item
+                </button>
+            </div>
+            {% if items is defined and items %}
+                <div class="float-right col-2">
+                    <form action="/items/delete" method="POST">
+                        <button type="submit" id="delete" class="btn btn-danger float-right">Delete All Items</button>
+                    </form>
+                </div>
+            {% endif %}
+        </div>
     </div>
-  </div>
-{% endblock %}
\ No newline at end of file
+    <div class="modal fade" id="modalAddItemForm" role="dialog">
+        <form action="/items" method="post">
+            <div class="modal-dialog modal-sm">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <h4 class="modal-title">Add Item</h4>
+                        <button type="button" class="close" data-dismiss="modal">&times;</button>
+                    </div>
+                    <div class="modal-body">
+                        <label for="title">Title:</label>
+                        <input type="text" id="title" name="title" class="form-control" placeholder="Enter title"
+                               autocomplete="off">
+                    </div>
+                    <div class="modal-footer">
+                        <button type="reset" class="btn btn-primary" data-dismiss="modal">Reset</button>
+                        <button type="submit" class="btn btn-primary">Create</button>
+                    </div>
+                </div>
+            </div>
+        </form>
+    </div>
+    <script type="text/javascript">
```

Comment:
As mentioned above, you can achieve the same results without the javascript below using submit buttons.